incognia_test.go

```diff
@@ -241,13 +242,28 @@ var (
 		PolicyID:       "policy-id",
 		Eval:           &shouldNotEval,
 	}
+	simplePaymentWebFixture = &Payment{
+		InstallationID: "installation-id",
```

Comment:
Web requests do not have installation id :eyes:

Reply:
True, this changes things a bit, cuz this value is not-optional.

Gonna make some changes to deal with this and I'll come back with a solution.